mappings/9a60e6300f4e9b34ad90f71ecc89be5d77aa00a437a5e1ae986ad4b24445414450584c5a.json

```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "9a60e6300f4e9b34ad90f71ecc89be5d77aa00a437a5e1ae986ad4b24445414450584c5a",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://twitter.com/buncathanhlong",
+        "signatures": [
+            {
+                "signature": "40052a45d3971e40e44708c7d982358dd78f89792648b6b444692a0e896236ca3cbab6bcf1adf5ea48e6f888bebc75ed1e2181a007fadd54dac3faba402d0908",
+                "publicKey": "5817526d712f71e33a31ac3429fb7ce70b3e17e727044d9a2a51493e7894ba48"
+            }
+        ]
+    },
+    "name": {
+        "sequenceNumber": 0,
+        "value": "DEADPXLZ",
+        "signatures": [
+            {
+                "signature": "9a39820f81ef439272d2e66488efffb1a81fd4965cb486dccdb7efa1095722598b1edde255a38198634bf23fc19ca98eebf6bb5897d9ec115b97bafdbcbd0502",
+                "publicKey": "5817526d712f71e33a31ac3429fb7ce70b3e17e727044d9a2a51493e7894ba48"
+            }
+        ]
+    },
+    "ticker": {
+        "sequenceNumber": 0,
+        "value": "DEADPXLZ",
+        "signatures": [
+            {
+                "signature": "79f94175569854e10e4f9ec2a4a5ba2de00dd18d174554603f5bc9a3e7b03f7a815dd3b337817f234a5731f614a3a29b7d2c3375d704cab7fc7fc76905d9d804",
+                "publicKey": "5817526d712f71e33a31ac3429fb7ce70b3e17e727044d9a2a51493e7894ba48"
+            }
+        ]
+    },
+    "decimals": {
+        "sequenceNumber": 0,
+        "value": 0,
+        "signatures": [
+            {
+                "signature": "225c69e0ad3b272a32405a207f14147069fc545c694d9c809d9ab632c450b7d58a67778dcf1e72ff1f8b49def6c789a11fdf5ec317b66488fbbd4e00f98c4c03",
+                "publicKey": "5817526d712f71e33a31ac3429fb7ce70b3e17e727044d9a2a51493e7894ba48"
+            }
+        ]
+    },
+    "policy": "820182018282051a06fe01c98200581c39a1df51147b6de6689a4727846962fb6540c3a3c7859a1a79b9420f",
+    "logo": {
+        "sequenceNumber": 0,
```

Comment:
Hi @Hywya,

the logo is from https://twitter.com/pxlzNFT
If this submission is connected to the project, please put policyID of the token to the twitter account or website.
Otherwise, please use a different logo

Thanks!